doc/rose-rosie-advice.html

```diff
@@ -0,0 +1,119 @@
+      This document is released under the <a href=
+      "http://www.nationalarchives.gov.uk/doc/open-government-licence/" rel=
+      "license">Open Government Licence</a>.<br />
+      <span id="rose-version"></span>
+    </address><img id="rose-icon" src="rose-icon.png" alt="Rose" />
+
+    <p><a href=".">Rose Documentation</a></p>
+  </div>
+
+  <div id="body-main">
+    <h1>Rose Micellaneous: rosie create v copy v checkout</h1>
+
+    <div id="content"></div>
+
+    <h2 id="rosie create"><var>rosie create</var></h2>
+
+    <h3>Description</h3>
+
+    <p>Allows the user to create a new suite and provides a new unique suite id 
+    which is provided by the roses repository. E.g.:</p>
+    <pre class="shell">
+rosie create
+</pre>
+
+    <h2 id="rosie copy"><var>rosie copy</var></h2>
+
+    <h3>Description</h3>
+
+    <p>Copies an existing suite using its <var>$SUITE_ID</var> from the roses 
+    repository to a unique suite id which is provided by the roses repository. 
+    E.g.:</p>
+    <pre class="shell">
+rosie copy $SUITE_ID
+</pre>
+
+    <p>The suite is assigned to the users, making them the owner unless they 
+    change this in the <var>rose-suite.info</var> file. 
+    The <var>rose-suite.info</var> is opened in an editor for the user to edit 
+    appropriately when the suite is being created. 
+    The <var>rose-suite.info</var> should provide information from the user to 
+    make the suite discoverable by themselves and others. 
+    Adding information to the <var>title</var> and <var>project</var> as 
+    required to make it meaningful and easily searchable using tools such 
+    as:</p>
+    <pre class="shell">
+    rosie go
+</pre>
+    <p>the suite browser GUI tool provided by rose.
+    or:</p>
+    <pre class="shell">
+    rosie ls
+</pre>
+    <p>the command line terminal search of all of the users suites in the 
+    roses repository.</p>
+
+    <h2 id="rosie checkout"><var>rosie checkout</var></h2>
+
+    <h3>Description</h3>
+
+    <p>Checks out an existing suite using its $SUITE_ID from the roses repository to your local roses/ directory using the original $SUITE_ID. E.g.:</p>
+    <pre class="shell">
+rosie checkout $SUITE_ID
+</pre>
+
+    <p>This suite is not a new unique copy and the original owner still owns 
+    the suite and therefore is the only person that can commit back to the 
+    trunk of that suite. 
+    However the owner can if desired add the person to the access list (in the 
+    <var>rose-suite.info</var> file) which then would give the <var>named</var> 
+    person full access to the suite. 
+    The suite owner may decided not to give access to other developers/users 
+    but can themselves merge others branch changes in to the trunk of their 
+    suite.</p>
+
+    <p>For developers/users that checkout a suite and want to branch from the 
+    suite the users should refer to their organisations version control system.
+    The Met Office uses fcm which is a wrapper to svn. E.g:</p>
```

Comment:
Remove Met Office reference. Instead, something like. "If you're using fcm as a wrapper to svn then..."